app/templates/events/view/settings.hbs

```diff
@@ -1,22 +1,41 @@
 <div class="row">
-  <p>
-    {{t 'Transfer ownership of this event to another user. You\'ll lose all the owner rights once they accept the ownership.'}}
-  </p>
-  <button {{action 'openEventTransferModal'}} class='ui red button'>
-    {{t 'Transfer Ownership'}}
-  </button>
-  <div class="ui divider"></div>
-  <p>
-    {{t 'Once you delete an event, there is no going back. Please be certain'}}
-  </p>
-  <button {{action 'openDeleteEventModal'}} class='ui red labeled icon button'>
-    <i class="trash icon"></i>
-    {{t 'Delete Event'}}
-  </button>
+	<div class="ui red segment">
+		<div class="ui grid">
+			<div class="ui left aligned eight wide column">
```

Comment:
Mixed tabs with spaces

Reply:
Fixed it, but CI should have failed for this.